src/common/Clock.h

```diff
@@ -11,14 +11,51 @@
  * Foundation.  See file COPYING.
  *
  */
-
 #ifndef CEPH_CLOCK_H
 #define CEPH_CLOCK_H
-
 #include "include/utime.h"
-
 #include <time.h>
 
-utime_t ceph_clock_now();
+struct ceph_clock {
+public:
+  ceph_clock() : start_(now_()) {}
+
+  utime_t now() {
+    return now_();
+  }
+
+  void reset() {
+    reset_();
+  }
+
+  utime_t elapsed() {
+    return elapsed_();
+  }
 
+private:
+  utime_t now_() {
+#if defined(__linux__)
```

Comment:
@shinobu-x 
Please modify this test to `if defined(LOCK_REALTIME)` 
So it'll work on all platforms haveing this const.

Reply:
@wjwithagen Thanks!